src/admin/migration.md

```diff
@@ -0,0 +1,194 @@
+
+- Please note that your `pgvector` extension is not installed in the `vectors` schema. This means that you can install `pgvector` and `PGVecto.rs` at the same time.
+
+### Steps to Migration
+
+#### 1. Install PGVecto.rs
+
+Please follow the [instructions](../getting-started/installation) provided to install `PGVecto.rs` and ensure that the extension is loaded by PostgreSQL.
+
+```sql
+CREATE EXTENSION vectors;
+```
+
+The two extensions, `pgvector` and `PGVecto.rs`, are now installed at the PostgreSQL.
+
+To validate the install, please run `\dx` in psql. You will see that there is an extension called `vector` for `pgvector` and another called `vectors` for `PGVecto.rs`.
+
+```shell
+postgres=# \dx
+                                                 List of installed extensions
+  Name   | Version |   Schema   |                                         Description                                          
+---------+---------+------------+----------------------------------------------------------------------------------------------
+ plpgsql | 1.0     | pg_catalog | PL/pgSQL procedural language
+ vector  | 0.7.2   | public     | vector data type and ivfflat and hnsw access methods
+ vectors | 0.3.0   | vectors    | vectors: Vector database plugin for Postgres, written in Rust, specifically designed for LLM
+```
+
+Please note that there are some symbols with the same name in both extensions. This may cause confusion when doing the migration. To differentiate between the two, please verify that the `search_path` does not include the `PGVecto.rs` schema `vectors`.
+
+```sql
+SHOW search_path;
+-- if it shows: `"$user", public` -> This is okay
+-- if it shows: `"$user", public, vectors` -> Remove `vectors` by:
+SET search_path = "$user, public";
+```
+
+#### 2. Check exist indexes
+
+For indexes created at vector columns, please retrieve the definition and record it in a convenient location. This information is necessary for the recovery of the indexes at a later stage.
+
+```sql
+postgres=# SELECT indexdef FROM pg_indexes WHERE tablename = 'items';
+                                       indexdef                                        
+---------------------------------------------------------------------------------------
+ CREATE UNIQUE INDEX items_pkey ON public.items USING btree (id)
+ CREATE INDEX items_embedding_idx ON public.items USING hnsw (embedding vector_l2_ops)
+(2 rows)
+```
+
+#### 3. Migrate vector columns
+
+To migrate from a pgvector vector column, the first step is to create a PGVecto.rs vector column of the corresponding type. The subsequent step is to replicate the vector data from the original column to the new one. 
+
+Here provides the conversion path for each vector type of `pgvector`:
+
+- `vector(pgvector) -> real[] -> vector(PGVecto.rs)`
+- `halfvec -> vector(pgvector) -> real[] -> vector(PGVecto.rs) -> vecf16` 
+- `sparsevec -> vector(pgvector) -> real[] -> vector(PGVecto.rs) -> svector`
```

Comment:
It's not good since casting from sparsevec to vector makes table much larger.

Reply:
Removed since sparse `svector` have a much larger dimension limit than dense `vector`. It is not possible to map from any sparse vector to a dense vector, regardless of in `pgvector` or `pgvecto.rs`.